### PR TITLE
Frost giants

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added Frost Giant cultural building.
- Added Frost Giant retinue.

### CODE REVIEW

File: localisation/000_wc_military_localization.csv [hunk 1]
```diff
@@ -343,6 +343,7 @@ wc_ret_cul_bloodtroll;War Mothers;;Kriegsm
 we_ret_cul_tuskarr;Turtle Cossacks;;Schildkr�tenkosaken;;;;;;;;;;;x
 wc_ret_cul_vrykul;Thanes;;Thanen;;;;;;;;;;;x
 wc_ret_cul_vulpera;Merchant Gunner;;Handelssch�tzen;;;;;;;;;;;x
+wc_ret_cul_frost_giant;Champions of Hodir;;;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/000_wc_military_localization.csv]: What if frost giants are not titanic?
For example, if they worship the old gods?

File: common/buildings/wc_castle_culture.txt [hunk 2]
```diff
@@ -5242,6 +5262,134 @@ castle = {
...
+				culture = frozen_giant
+			}
+		}
+		trigger = { 
+			TECH_CASTLE_CONSTRUCTION = 4
+			ROOT = {
+				culture = frozen_giant
+			}
+		}
+		upgrades_from = ca_culture_ogre_3
```
Comment (MemeWizard) [inline common/buildings/wc_castle_culture.txt]: 🤔

Comment (sejtalk) [reply common/buildings/wc_castle_culture.txt]: fixed

File: localisation/000_wc_military_localization.csv (hunk 1 above)
Comment (sejtalk) [reply localisation/000_wc_military_localization.csv]: changed to more neutral

**Comment by MemeWizard** [comment]
@sejtalk, changed triggers a bit in 8d1c150, so troll and titanforged buildings now have similar code structure.
Also, moved titanforged buildings from dwarf group to titanforged group in 
b569296.

**Pull request merged**